<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color:#F1f1ef;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame {
background-color:#Dbdcd8;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="h1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#h1{
color:#484848;
font-family:sans-serif;
font-size:22px;
}</string>
     </property>
     <property name="text">
      <string>Agenda</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="txtDt">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>10</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#txtDt{

}</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>4</month>
       <day>19</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
     <property name="date">
      <date>
       <year>2024</year>
       <month>4</month>
       <day>19</day>
      </date>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameHoje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameHoje{
background-color:#fde685;
border-radius:5px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCalendario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagens/cedula.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalHoje">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>47</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblTotalHoje{
color:#484848;
font-family:sans-serif;
font-size:40px;
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblHoje">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblHoje {
color:#484848;
font-family:sans-serif;
font-size:14px;
}</string>
     </property>
     <property name="text">
      <string>Hoje</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVisualizarHoje">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnVisualizarHoje{
background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Visualizar lista</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameTodos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameTodos{
background-color:#Dbdcd8;
border-radius:5px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCalendario_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagens/livros.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalTodos">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>47</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblTotalTodos{
color:#484848;
font-family:sans-serif;
font-size:40px;
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTodos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblTodos {
color:#484848;
font-family:sans-serif;
font-size:14px;
}</string>
     </property>
     <property name="text">
      <string>Todos</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameConcluidos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameConcluidos{
background-color:#Dbdcd8;
border-radius:5px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCalendario_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagens/lapis-regua.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalConcluido">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>47</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblTotalConcluido{
color:#484848;
font-family:sans-serif;
font-size:40px;
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblConcluido">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblConcluido{
color:#484848;
font-family:sans-serif;
font-size:14px;
}</string>
     </property>
     <property name="text">
      <string>Concluidos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVisualizarConcluido">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnVisualizarConcluido{
background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Visualizar lista</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNovo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnNovo{
color:#484848;
font-family:sans-serif;
}</string>
    </property>
    <property name="text">
     <string>+ Nova Tarefa</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImg">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblImg{
border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/Portapapeles plano aplanado calendario maqueta _ Archivo PSD Premium.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frameProgramado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>271</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#frameProgramado{
background-color:#Dbdcd8;
border-radius:5px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblCalendario_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../imagens/calendario-linhas-caneta.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalProgramado">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>47</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblTotalProgramado{
color:#484848;
font-family:sans-serif;
font-size:40px;
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblProgramados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lblProgramados
{
color:#484848;
font-family:sans-serif;
font-size:14px;
}</string>
     </property>
     <property name="text">
      <string>Programados</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVisualizarProgramados">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnVisualizarProgramados{
background-color:transparent;
}</string>
     </property>
     <property name="text">
      <string>Visualizar lista</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
